static/flow/style.css

```diff
@@ -925,19 +925,26 @@ button.filter {
 @font-face {
     font-family: 'museo500';
     src: url('fonts/museo500-regular-webfont.woff2') format('woff2'),
-         url('fonts/museo500-regular-webfont.woff') format('woff');
```

Comment:
This whole style sheet needs clean-up work.  I started in on some of it before your rotation started, but we could greatly improve many things in here.